src/applications/ivc-champva/10-7959C/components/MissingFileConsentPage.jsx

```diff
@@ -16,6 +16,7 @@ export function MissingFileConsentPage(props) {
     showConsent: true,
     fileNameMap: { ...requiredFiles },
     requiredFiles,
+    nonListNameKey: 'applicantName',
```

Comment:
Leveraging the new `nonListNameKey` property that lets us designate an arbitrary property as the name we want to display on the `MissingFileOverview` page